packages/components/README.md

```diff
@@ -0,0 +1,23 @@
+# @288-toolkit/components
```

Comment:
Thats not a package, you can remove this one.

Reply:
👍  See 40db7702f7